.github/workflows/smoke-test.yml

```diff
@@ -242,7 +274,11 @@ jobs:
       - uses: r-lib/actions/setup-r-dependencies@v2
         with:
           working-directory: R/opendp
-          extra-packages: devtools, local::., RcppTOML, lintr
+          extra-packages: devtools, RcppTOML, lintr
+
+      - name: Install OpenDP
+        working-directory: R/opendp/
+        run: Rscript -e 'devtools::install()'
```

Comment:
This replaces `local::.`? Short note relating this to library change wouldn't hurt.